### DISCUSSION TITLE
Move plNetLog tool into Plasma repo

### DISCUSSION THREAD

**Discussion by dpogue** [H-uru/Plasma#1084] [Ideas & Proposals]
So https://github.com/H-uru/plNetLog was written to inspect network traffic from the H-uru/Plasma internal client, but it hasn't really been maintained over the years. It appears to only require the Qt framework as a dependency.

I'm proposing to import plNetLog into the Sources/Tools folder of this repo, and make any necessary updates for it to build with the version of Qt being used by the other tools. This will provide easier access to it (by having it built automatically as part of CI) and encourage it to be kept maintained along with the rest of the engine.

Eventually, once the netcode in Plasma is no longer Windows-specific, I'd like to explore options for a *nix implementation of plNetLog.

**Comment by Deledrius** [comment]
Sounds like a great idea to me.

**Comment by Hoikas** [comment]
Let's make it happen.

**Comment by zrax** [comment]
It looks like I wrote the original, so I'll see about porting it to Qt5 and putting it in this repo...  Actually using the data structures and code in Plasma will probably come as a later step, as right now it does all of its own I/O and processing.
